features/step_definitions/article_steps.rb

```diff
@@ -150,7 +150,7 @@
 
 Then /I should see admin user comment on (.+) page/ do |page|
   page.on do
-    expect(page.given.text).to include(Howitzer.app_test_user)
+    expect(page.given.text).to include(out(:@user).name.upcase)
```

Comment:
it is related with previous comment about admin user. So, I would use Admin user name as a static text here

Reply:
Previously admin email was used in comment, but now it's a name. Admin user as far as i understand identified with email, and name is new each time. So its impossible to use static name